src/utils/HttpErrorHandler.ts

```
@@ -0,0 +1,55 @@
+import { AxiosError } from 'axios'
+
+export interface ErrorResponse {
+  message: string
+  status: number
+  data?: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)